tests/test_transform.py

```diff
@@ -119,6 +120,24 @@ def test_process_zipfile_failed_pdf(self, fake_decapitate):
         # clean the test directories
         clean_test_directories()
 
+
+    @patch.object(transform, 'decapitate_pdf_with_error_check')
+    def test_copy_pdf_to_output_dir_timed_out_pdf(self, fake_decapitate):
+        "tests of when pdf decapitaion reaches timeout"
+        zipfile_name = os.path.join(TEST_DATA_PATH,
+                                    '18022_1_supp_mat_highwire_zip_268991_x75s4v.zip')
+        file_title_map = {
+            '18022_1_merged_1463214271.pdf': 'Merged PDF'
+        }
+        fake_decapitate.side_effect = FunctionTimedOut
+        doi = '10.7554/eLife.12717'
+        with zipfile.ZipFile(zipfile_name, 'r') as current_zipfile:
+            with self.assertRaises(Exception) as context:
+                return_value = transform.copy_pdf_to_output_dir(file_title_map, None, doi,
+                                                                current_zipfile, POA_CONFIG)
+        clean_test_directories()
```

Comment:
this `clean_test_directories()` should still go in `tearDown()`, to guarantee it will always be executed

Reply:
I think it is already in the ``tearDown()`` :+1:

Reply:
Yes, I see it is now. So it can probably be removed from the test methods themselves in a follow-up, since there's no need to execute it twice?

Reply:
I was not aware ``tearDown()`` is called at the end of each method of the class, thinking it was only called when the object is destroyed. I've removed the duplicates and also converted an open zip file to a ``with`` statement.

Reply:
have to cross-check that because I could be confusing Pytest/unittest with PHPUnit, but every test method should use a brand new object instantiated from the test class

Reply:
I tested it a couple different ways here, and it indicated the ``tearDown()`` was called after each method. There's at least one test here that will fail if the directories are not cleaned, and it passes with ``clean_test_directories()`` only present in ``tearDown()``.